src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@janvorli The first approach (revise ``OOPStackUnwinderX86::VirtualUnwind`` to update ``ESP`` as ``Caller-SP``) will make CLR get into stuck when resuming. Thus, we need to additionally maintain SP for resume.

I'm not sure whether it is possible to take the second approach. The discussion in #9272 shows that libunwind does not consider calling convention for unwinding. As the issue happens when we unwind native method frame, we could not use the existing infrastructure for managed methods.